crates/sui-types/src/sui_system_state.rs

```diff
@@ -192,4 +192,13 @@ impl SuiSystemState {
             net_addresses,
         }
     }
+
+    pub fn get_current_epoch_narwhal_committee(&self) -> NarwhalCommittee {
+        // todo: we need the following to be added to the Validator Metadata
+        // todo: on the system state object in order to construct this:
+        // - protocol_key
+        // - network_key
+        // - narwhal_primary_address
+        todo!();
```

Comment:
As I wrote on the checkpoint/reconfig slack channel , depending on how much effort this is we can build this incrementally and attempt to build the Narwhal committee based on the genesis committee - as for Wave 2 we are not looking to change the validator set (or change the addresses/keys).